.github/workflows/update-helm-repo.yaml

```diff
@@ -96,26 +99,39 @@ jobs:
     needs: [setup]
     runs-on: ubuntu-latest
     permissions:
+      id-token: write # allows GitHub App to generate id-token from Github's OIDC
       contents: write # allows GITHUB_TOKEN to push chart release, create release, and push tags to github
       packages: write # allows GITHUB_TOKEN to push package to ghcr
     env:
-      github_app_id: ${{ secrets.github_app_id }}
+      GITHUB_APP_ID: ${{ secrets.github_app_id }}
```

Comment:
I think the PRIVATE_KEY initial value should be the in coming secrets.github_app_pem

Reply:
You're right, I've missed that. Fixed now.